src/foraging_gui/bias_indicator.py

```diff
@@ -98,7 +98,7 @@ def x_range(self, value: int) -> None:
         self._x_range = value
 
     def calculate_bias(self,
-                       time_point: float,
+                       trial_num: int,
                        choice_history: Union[List, np.ndarray],
                        reward_history: Union[List, np.ndarray],
                        n_trial_back: int = 15,
```

Comment:
set n_trial_back to 5

Reply:
The default should be 5? I was using 15 based on the default of the fit_logistic_regression function. Easy and happy to change though

Reply:
Yeah, based on some investigating I did this week, we should use 5